### DISCUSSION TITLE
🍇 Fruit

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1949] [Ideas]
**[DRAFT] - This is an in progress team draft, values are not finalised**

The purpose of this proposal is to introduce the fruit feature, gather feedback and improve.

This discussion follows the new format that the team will embark on for 2023 development.

1. Open Proposal with desired mechanics
2. Collaborate with community to improve design/playability = update proposal
3. Solidify details in our docs.sunflower-land.com
4. Build feature
5. Open up early testing for feature
6. Tweak and improve
7. Launch for public.

## Fruit

The purpose of fruit is to provide a new farming mechanism to earn SFL in the game. It follows a similar gameplay mechanic to crops so it can be understood easily, but adds a few variations for added interest.

**Goal of Fruit**

- Provide bonus avenue for advanced players to earn SFL
- Add variation for high-end cooking recipes. Quick cooking (30 minutes), but mid-level XP and Grub Shop SFL
- Introduce variation in plant growth gameplay
- Engagement through building progression and quests.

## Details

You can buy fruit seeds at the Market and plant them on the special fruit patches. The aim is to harvest fruit so that you can:

- Sell it at the Market for SFL
- Make food and level up your Bumpkin
- Make food and sell it at the Grub Shop
- Complete Bumpkin Quests and mint wearables

## Fruit Patches

Fruit seeds can only be placed on 'Fruit Patches' - Fruit patches appear on your 10th piece of land with a pre-grown Apple Tree.

<img width="203" alt="Screen Shot 2022-12-29 at 2 44 23 pm" src="https://user-images.githubusercontent.com/11745561/209900642-7bbce0bd-b5cb-4a88-9233-e7f9a9194f97.png">

## Planting & Harvesting 

![grass_patch_empty](https://user-images.githubusercontent.com/11745561/209901628-3fd06796-421a-4525-8632-c02503e5cfde.png)

When you plant a fruit seed, you must wait a period of time before you can harvest the fruit.

Unlike crops, fruit can be harvested multiple times before the plant dies. This means that once a Fruit has grown, it can be harvested, the player waits and then it can be harvested again.

Each fruit can be harvested between **3-5 times**. This is randomly set each time and you won't know how many harvests a fruit has left.

Once you have fully harvested the fruit, it will disappear and you can plant a new fruit seed.

### Fruits

_Don't forget, one seed produces 3-5 harvests worth of fruit._

**Blueberry**

![blueberry_bush](https://user-images.githubusercontent.com/11745561/209901003-c86aea26-71d0-4002-adbb-0f39398bf0cb.png)

- Unlock at Lvl 13
- Seed cost - 0.375 SFL
- Fruit sale price = 0.15 SFL
- Growth time = 12 hours
- Replenish time (between harvests) = 4 hours

**Orange** 

![orange_tree](https://user-images.githubusercontent.com/11745561/209901638-ed89e32c-3796-4455-b0ce-12a3ea84a4d9.png)

- Unlocks at Lvl 14
- Seed Cost - 0.625 SFL
- Fruit sale price = 0.225 SFL
- Growth time = 12 hours
- Replenish time (between harvests) = 6 hours

**Apple**

![apple_tree](https://user-images.githubusercontent.com/11745561/209901646-01fd9a2f-b89f-40f9-abf6-8b50cecabc69.png)

- Unlocks at Lvl 15
- Seed Cost = 0.875 SFL
- Fruit sale price = 0.3125 SFL
- Growth time = 12 hours
- Replenish time (between harvests) = 12 hours

## Smoothie Stand

![smoothie-stand](https://user-images.githubusercontent.com/11745561/209902363-8a9f183b-8f85-4651-84e6-8a0f02ad778f.png)

Once you reach lvl 15, you will be able to build the Smoothie Stand. The purpose of the smoothie stand is to prepare quick meals that produce mid level XP and Grub Shop SFL.

The requirements are 25 Wood, 25 Stone and 10 Iron. This is estimated to take 2-3 days of gameplay at level 15 to acquire.

**Orange Juice**

- 5 Oranges
- 30 minutes cooking time
- 300 XP
- 2.5 SFL at Grub Shop

**Apple Juice**

- 5 Apples
- 30 minutes cooking time
- 420 XP
- 3.5 SFL at Grub Shop

**Power Smoothie**

- 5 Kale, 10 Blueberries
- 30 minutes cooking time
- 600 XP
- 5.0 SFL at Grub Shop

**Bumpkin Detox**

- 5 Apples, 5 Oranges, 10 Carrots
- 30 minutes cooking time
- 800 XP
- 6.3 SFL at Grub Shop

**Purple Smoothie**

- 10 Cabbage, 5 Blueberry
- 30 minutes cooking time
- 250 XP
- 2.1 SFL at Grub Shop

## Bumpkin Fruit Quests

_Ideation stage - placeholder ideas_

There will be multiple wearables that you can mint for free simply by completing some fruit picking challenges.

_Estimated times are based on having 5 fruit patches. Players will likely have between 3-10 patches based on their progress_

**Wearable Shirt**

- Purpose: Cosmetic
- Harvest 10 Blueberries
- Estimated time = 1 day

**Wearable Pants**

- Purpose: Cosmetic
- Harvest 100 Oranges
- Estimated time = 8 days

**Wearable Hat**

- Purpose: Utility - Bonus 0.25 Fruit on each harvest
- Harvest 200 Apples
- Estimated time = 1 month

**Wearable Tool**

- Purpose: Cosmetic
- Harvest 1200 Blueberries
- Estimated time = 2-3 months

## Bumpkin SFTs

_Ideation stage - placeholder ideas_

**Immortal Pear**

In Chinese Mythology, Pears are a symbol of immortality.

Boost: Fruit Trees persist for 1 bonus harvest
Supply: Unlimited
Estimated SFL Burn - 110.5 SFL
Ingredients: 25 Gold

**Lady Bugs**

Lady Bugs are a natural fight against harmful fruit tree chemicals. 

Boost: 25% chance of a 2x critical Apple Drop
Supply: 2500
Estimated SFL Burn: 141.75 SFL
Ingredients: 25 Gold, 100 Apples
Estimated Collection Time: 13 days

**Squirrel Monkey**

A natural orange predator. Orange Trees are scared when a Squirrel Monkey is around, grows twice as fast.

Boost: Orange Trees grow 2x fast
Supply: 1000
Estimated SFL Burn: 178 SFL
Ingredients: 25 Gold, 300 Oranges
Estimated Collection Time: 22.5 Days

**Black Bearry**

This bear needs vast amounts of blueberries to survive. Blueberry trees produce 2x.

Boost: 2x Blueberry yield
Supply: 500
Estimated SFL Burn: 215.5 SFL
Ingredients: 25 Gold, 700 Blueberries
Estimated Collection Time: 41 Days

**Comment by Polysemouse** [comment]
I would recommend that different ingredients are used to craft Power Smoothie and Bumpkin Detox, given that the XP and SFL rewards at the Grub Shop are different

**Reply by adamhannigan** [reply]
Good catch. That was a typo, they do have different resources. Just updated ✅ 

The Detox is an Apple + Orange + Carrot mixture

**Comment by EJeer** [comment]
Orange seed and fruit price is out of proportion compared to others.
For others, rations are better than 3:1, which means 3 fruits give more SFL than the costs of seed, but the orange ratio is less than 3:1, meaning you need to harvest at least 4 fruits to cover seed costs.

**Reply by FarmerKuenster** [reply]
orange seed gives you 0.675 total for 3 harvests.  So a profit of 0.05 SFL each time. So seed cost is covered, but not by a lot. Hope this helps! :)

**Comment by Formalinfarmer** [comment]


**Reply by adamhannigan** [reply]
Thanks for your feedback.

Yes, the idea of the Bumpkins is to introduce a new Quest mechanic. You must collect X fruits to mint free pieces of Bumpkin Wearables. Some of these wearables will add utility to your game 😄 

There will also be available rare SFTs to work towards 🚀 

Details of these will be added soon.

**Reply by Formalinfarmer** [reply]
I deleted my comment by accident :(  thanks for the response :)

**Comment by celpower** [comment]
Power Smothie and Bumpkin Detox have same recipe. I think Carrot should be considered in a recipe or new recipe. So far love the idea, I will just need to wait to go live to see the XP/Cost/Sale at Grub shop is really worth the production of these smothies or if it will be one of the few i only do to sell at Grub shop and not really use as a source of XP

**Reply by adamhannigan** [reply]
Thanks, Fixed ✅

**Comment by tryjapa** [comment]
There is any actual bonus that will affect fruits?

**Reply by adamhannigan** [reply]
Yes, there will be Bumpkin Wearables + SFTs. Details coming soon.

**Comment by KaKaDoVieira** [comment]
It would be nice if we get 1 wood when the tree goes down in the last harvest. It will be more appeal. And maybe people will be more quiet about axe problem... (kill two rabbits with one shot). Max people can do its 30 wood more buffs, if we have a stock of 10 each. And it will take a long time, so people will probably sync before that. Any way its just a idea 🙂

**Reply by adamhannigan** [reply]
Agreed. This was being discussed amongst the team - the current implementation on testnet does include a free piece of wood when removing the 'Dead Tree'

**Comment by vrillx** [comment]
Why are we stuck on all food to give XP only ? This could be a so much greater experience . The variety of bonuses food could have is endless. 

We could have food that gives 

- a small xp gain for 10 mins (5 or 10%)
- a small increase to harvesting certain crops for 5 mins
- expand the life of our tools for 5 mins
- reduce a node timer by 30 mins ( when used on it)
- reduce a node respawn timer by 60 mins ( when used on it)
and on and on .... 

Lets expand fruits beyond the regular 200 xp when eating them...

**Reply by adamhannigan** [reply]
I believe this is touching on the plans for fertilisers/potions.

These will require a mixture of crops, fruits & mushrooms to provide temporary buffs to a farm.

**Reply by vrillx** [reply]
ok glad there is a plan for buffs also

**Comment by NomiceBinocle** [comment]
I love this new mechanic can't wait to be able to farm it's new crops 

I have a question:
will the scarecrow buffs (t1, t2 and t3) be effective on fruits?

**Reply by adamhannigan** [reply]
T1 & T2 Scarecrows will only affect the current crops in the game.

T3 will affect all seeds, which includes Fruit Seeds.

**Reply by NomiceBinocle** [reply]
thank you for the details

**Comment by IamSooy** [comment]
Regarding the Bumpkin SFTs

-Would it be better if there was no limited supply for the Bumpkin SFTs until a specific date (reasonable and balanced) 
-This would balance those people who have reached land 11 and people who are only at land 7 and encourage new people to grind.

Mechanics:
*Supplies:
	Immortal Pear	      -3000
	Lady Bugs	      -2500
	Squirrel Monkey   -1000
	Black Bearry	     -500

But here is the Catch: 

**The supply is unlimited until a specific date (reasonable and balanced).
Scenarios:
		Scenario 1: I want to focus on minting Black Bearry SFTs, which will be tough (reasonable). This will cause you to be unable to craft the other 3 Bumpkin SFTs.
		Scenario 2: I will craft the Immortal Pear Because it is much easier; I could also craft Lady Bugs or Squirrel Monkey if I grind more.


**When the specific date (reasonable and balanced) ends, and the supplies are not yet sold out. An exact date will be announced when it will be live again. Same with the other Bumpkin SFTs that are not sold out.

**However, if the crafted are more than the designated supply, it will no longer open/live again.

Thank you.

**Reply by adamhannigan** [reply]
That is a nice/fair idea. Unfortunately this puts us at risk of minting too many items which would have a negative effect on the overall game economy.

The max supply of `500` was calculated to have less than 5% impact on the overall fruit production. Anything that goes above 5% is to OP and can shift our economy.

If we were to do a time based mint, we would need to figure out a large amount of blueberries that means only 500 people would reach it - (2000-3000 blueberries).

Players definitely have a choice and need strategy when targeting these SFTs - do they focus on Blueberries, Apples or Oranges? 😱

**Reply by IamSooy** [reply]
I see; I understand. Thank you for your reply.

Question: 
	In your calculation, Do you have the data of the players who have reached land 8-11, and if so, do you think the 500 supply is enough for them and have some extra for the players who are only in land 7? If not, I have another mechanics proposal.
 
Another proposal regarding the mechanics of Bumpkin SFTs.

	Since we will stick to the supply, I suggest we should spread it. To be fair and balanced. 

IT CAN BE CHANGED CONCERNING HOW MANY PLAYERS ARE IN THIS LANDS
Black Bearry (500)
	Land 11-100
	Land 10-100
	Land 9-100
	Land 8-100
	Land 7-100
		Total: 500 Supply

Goal: Somehow lessen the enormous gap between the older and new players.

Thank you.

**Reply by adamhannigan** [reply]
These SFTs will be targeted at players lvl 13+, since that is the level needed to gather fruit.

I think there will be roughly 5000-6000 people that would like a Black Bear, but only 500 will be able to craft it. 

Unfortunately not everyone will be able to mint this item, but there will be other items that players can mint (e.g. Immortal Pear that has unlimited supply)

**Reply by IamSooy** [reply]
I see; I understand. Thank you for your reply.

Yes, These SFTs will target players lvl 13+, but it will be useless if they are not on land 7-11 right? Since only at land 7 and above you can you plant the seed tree. And the higher your land is the more you can plant it, which is why I propose the later mechanics to balance it out.

Oh Immortal Pear will be an unlimited supply? Above it says 3000 supplies... is that final?

Thank you.

**Reply by adamhannigan** [reply]
Good catch on the Immortal Pear - I just updated that 👍 

Essentially these SFTs are to be considered a special event. These are targeted at players who have been around for 2-3 months.

**Reply by dragonhunter2017** [reply]
May be we can introduce a quota system where 5%  of the NFT is secured for the new player (those who joined after LE, may be identified by land number). Then new players can compete within themselves. This 5% can be varied depending on the number of new players and planned NFT release. (btw, I am an old player. Don't know if it is possible to implement the idea in real )

**Comment by popalexandrudan** [comment]
I feel like the race for this NFT's is not fair based on what i can see on SFL Data.

Some players with 11 expansions have **3 fruit patches,** while some have **6 fruit patches**.

Isn't the RNG difference way to big? In my opinion, everybody with 11 expansions should have the same chance, while expanding after 11, could be more in RNG hands.

I would understand 4 fruit patches vs 5 fruit patches, but 3vs6 is to much.

**Reply by IamSooy** [reply]
This is my concern that's why I have proposed a mechanics to be balanced this out and be fair.

The gap just gets bigger between new and old players if this continues.

**Reply by adamhannigan** [reply]
We will work on balancing this out on the following Land expansions which will become available.

Those with lesser fruit patches will have a higher chance of finding patches

**Reply by pappajan69** [reply]
there is only 400 players that max out there land so far and most of them did get big head start and to bring more land now will make it even harder for small players to keep up.                                                                                                                                                                               so every one that got land max will get that 500 bears fast so                                                                                                                                              for the rest of us getting gold take for ever and the players that have max land have all the best nfts                                                                so if u have 3 fruit patches or 6 u can make it work .

**Comment by Telknub** [comment]
Will there be a skill tree for fruits? 

Since crop buffs does not work on fruits, it would make sense to make a skill tree for fruits.

The same would make sense for other future features, like fishing skills, foraging skills (for mushrooms?), beekeeping skills(maybe they can be on animals), treasure hunter skills (maybe exploration skills, with other skills related to other islands, since each island has its own purpose), etc.

**Reply by adamhannigan** [reply]
Yes, we will add skills for fruits.

This likely won't be done on launch, but at a later date 👍

**Comment by FarmerKuenster** [comment]
Hey @adamhannigan any chance there will be a squirrel monkey NFT art request? I made a little mock up and I'll add it here for now. 

Squirrel monkey statue to keep the orange trees scared and produce more :)

![squirrel monkey statue NFT-export](https://user-images.githubusercontent.com/104322749/211165055-88fad663-56d6-40a7-aa6a-1dba4ee13660.png)

**Reply by adamhannigan** [reply]
This is cute! I will follow up with the art contractors, they are working on this at the moment 🙏

**Comment by future2back** [comment]
Bumpkin SFTs are not reachable for everyone, thats not good.

The game always tries to strengthen the strongest. You are creating a monster and not realizing it. How do you expect new players?

If releases replace old items, it might even be different, but that's not how it happens.



The game does not encourage new players.

**Reply by IamSooy** [reply]
Totally True.

**Reply by future2back** [reply]
> T1 & T2 Scarecrows will only affect the current crops in the game.
> 
> T3 will affect all seeds, which includes Fruit Seeds.


This is crazy. Devs are missing the opportunity to create new NFTS and better distribute the game.

New player = No opportunity.

Very frustrating for the little ones who have been playing for a while, imagine who starts now?


You should rethink this NFT/Boots distribution.

**Reply by adamhannigan** [reply]
Thanks for your feedback @future2back 

The fruit release introduces the Helios Blacksmith. This provides items that have unlimited supply for all players to craft. The first of the items is the Immortal pear which has an unlimited supply.

The goal is to ensure new players have something to work towards. This is the first of many craftable items to come thanks to this feature 🚀

**Reply by future2back** [reply]
@adamhannigan Thanks for replying.

What is available in Helios Blacksmith favors strongest as well, its easy to them.

We are talking about what is available on Goblin Craft. It is not achievable for new players(active).

If this is always the case, it will always be frustrating for those who are not big. On top of all that, it doesn't seem sustainable at all.

You guys should rethink the distribution of boots within the game. Why does the big one have to have it all? They must have enough resources to buy from the weakest. Will the weakest never have anything that interests the strongest?

**Reply by adamhannigan** [reply]
@future2back Like most games, you need to play for 3-4 months before you can obtain a high-end game.

We will provide items that are easy for newcomers and also provide challenging items which engage our long term players.

At the end of the day we won't cater any group, we cycle and mix the types of items we provide whether they are free, special event, limited supply, require lots of time, resources etc.

**Comment by IvanSkazkin** [comment]
I wanted to say that it is correct that you moved the release and released the update without errors. This is very valuable, thank you. The quality is felt.

**Reply by adamhannigan** [reply]
Thanks! We definitely feel this format is more suitable, especially the allocated time for beta testers

**Comment by Xeko-92** [comment]
Can you confirm if the squirrel monkey works on the replenish harvests too? Or if it's only the initial tree growth?

**Comment by vitt0** [comment]
Is the Black Bearry going to give access to the Vip role in Discord?
